packages/pipeline-editor/src/utils.ts

```diff
@@ -33,27 +32,34 @@ export default class Utils {
    */
   static generateNotebookPipeline(
     filename: string,
-    options: ISubmitNotebookOptions
+    runtime_config: string,
+    framework: string,
+    dependencies: string[],
+    envObject: { [key: string]: string }
   ): any {
     const template = JSON.parse(JSON.stringify(pipeline_template));
     const generated_uuid: string = Utils.getUUID();
 
     const artifactFileName = filename.replace(/^.*[\\/]/, '');
     const artifactName = artifactFileName.replace(/\.[^/.]+$/, '');
 
+    const envArray = Object.entries(envObject).map(
+      ([key, val]) => `${key}=${val}`
+    );
+
     template.id = generated_uuid;
     template.primary_pipeline = generated_uuid;
     template.pipelines[0].id = generated_uuid;
 
     template.pipelines[0].nodes[0].id = generated_uuid;
     template.pipelines[0].nodes[0].app_data.filename = filename;
-    template.pipelines[0].nodes[0].app_data.runtime_image = options.framework;
-    template.pipelines[0].nodes[0].app_data.env_vars = options.env;
-    template.pipelines[0].nodes[0].app_data.dependencies = options.dependencies;
+    template.pipelines[0].nodes[0].app_data.runtime_image = framework;
+    template.pipelines[0].nodes[0].app_data.env_vars = envArray;
+    template.pipelines[0].nodes[0].app_data.dependencies = dependencies;
 
     template.pipelines[0].app_data.name = artifactName;
     template.pipelines[0].app_data.runtime = 'kfp';
```

Comment:
You might need to rebase here, I believe we removed the hard code in master.

Reply:
As a follow up to this, I believe we recently touched other code you've moved to different locations as well. I would double check changes to files this PR "deletes"